client/src/GlobalStyle.js

```diff
@@ -0,0 +1,19 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
You will get better logs and developer experience if you use [babel-macro](https://styled-components.com/docs/tooling#babel-macro):

```suggestion
import { createGlobalStyle } from "styled-components/macro";
```

Reply:
Thanks